### PR TITLE
panel meeting search pagination TM-4024

### PR DESCRIPTION
**Pull request by SidelineCory24**
😺
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2470)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/762)

To-do
- [x] return total results/count for panel meeting search

### CODE REVIEW

File: src/services/agendas.js [hunk 1]
```diff
@@ -373,63 +373,94 @@ const getPanels = async (filsCols, query) => {
...
+          'panelmeetingdatetypes.mdtdesctext',
+          'panelmeetingdatetypes.mdtordernum')
+        let filterTable = {
+          'pmseqnum': 'panelmeetings.pmseqnum',
+          'pmscode': 'panelmeetings.pmscode',
+          'pmpmtcode': 'panelmeetings.pmpmtcode',
+        };
+        filsCols['filters'].map(fc => {
+          return qb.where(filterTable[fc.name], fc.method, fc.value);
+        })
```
Comment (scott062) [inline src/services/agendas.js:428]: Make the code smell a reusable callback so the identical qb is only used once

File: src/services/agendas.js [hunk 2]
```diff
@@ -373,63 +373,94 @@ const getPanels = async (filsCols, query) => {
...
```
Comment (scott062) [inline src/services/agendas.js:399]: See other comment

**Comment by elizabeth-jimenez** [comment]
Approved Pending Feedback: after addressing Scott's comment

**Review by elizabeth-jimenez — APPROVED** [review]
🤌

File: src/services/agendas.js (hunk 1 above)
Comment (SidelineCory24) [reply src/services/agendas.js]: optimization has been pushed

**Review by scott062 — APPROVED** [review]

**Pull request merged**